src/axom/quest/SamplingShaper.hpp

```diff
@@ -771,7 +786,7 @@ class SamplingShaper : public Shaper
 
       AXOM_ANNOTATE_BEGIN("fct project");
       axom::for_all<axom::SEQ_EXEC>(0, NE, [=](int i) {
-        shaping::FCT_project(&m_d(0, 0, i),
+        shaping::FCT_correct(&m_d(0, 0, i),
```

Comment:
I renamed this function to `FCT_correct` since the projection happens in the `BatchLUSolve` a few lines up, and this function applies the FCT correction.

(Previously, this function applied the projection and correction).